recipe/meta.yaml

```diff
@@ -24,20 +25,22 @@ requirements:
     - setuptools
     - blas 1.1 {{ variant }}
     - openblas 0.2.20|0.2.20.*
```

Comment:
We can drop the `openblas` pinning and just leave the dependency.

Reply:
Done